src/inspector_profiler.cc

```diff
@@ -218,6 +224,45 @@ void V8CoverageConnection::End() {
   CHECK_EQ(ending_, false);
   ending_ = true;
   DispatchMessage("Profiler.takePreciseCoverage");
+  WriteSourceMapCache();
+}
+
+void V8CoverageConnection::WriteSourceMapCache() {
+  Isolate* isolate = env_->isolate();
+  Local<Context> context = env_->context();
+  HandleScope handle_scope(isolate);
+  Context::Scope context_scope(context);
+
+  Local<Function> source_map_cache_getter = env_->source_map_cache_getter();
+  Local<Value> source_map_cache_v;
+  Local<Value> argv[0] = {};
+  if (!source_map_cache_getter->Call(env()->context(), Undefined(isolate), 0, argv)
```

Comment:
What's the second argument to `V8::Function`, in the docs I found online I only found three arguments listed:

https://v8docs.nodesource.com/node-0.8/d5/d54/classv8_1_1_function.html#a111b53439348e8726a5100ca5619484c

I cribbed `Undefined(isolate)` from elsewhere in the codebase.

Reply:
It’s the `this` object for the function call – I’m not sure if there’s public generated HTML API docs out there that are up to date for V8, but in the worst case, the inline docs in deps/v8/include/v8.h are mostly helpful.

Reply:
:+1: sounds like creating an `Undefined` this, is probably a reasonable call then?